graphene_django_extras/fields.py

```diff
@@ -221,7 +221,8 @@ def __init__(
         if not kwargs.get("description", None):
             kwargs["description"] = "{} list".format(_type._meta.model.__name__)
 
-        kwargs.pop('resolve_queryset')
+        if kwargs.get('resolve_queryset'):
```

Comment:
why is this change? if you are trying to check if the key `resolve_queryset` is present before popping then this will throw exception

Reply:
Removing this....it was temp change for one of the model where I didn't inherit Graphql